[2025-05-30T01:23:21.673+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T01:23:21.863+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T00:56:00+00:00 [queued]>
[2025-05-30T01:23:21.931+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T00:56:00+00:00 [queued]>
[2025-05-30T01:23:21.932+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-30T01:23:22.143+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): execute_task> on 2025-05-29 00:56:00+00:00
[2025-05-30T01:23:22.191+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1070) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-30T01:23:22.201+0000] {standard_task_runner.py:72} INFO - Started process 1076 to run task
[2025-05-30T01:23:22.187+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flag_check_mysql_dag', 'execute_task', 'scheduled__2025-05-29T00:56:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/Sesion3/Ejemplo7.py', '--cfg-path', '/tmp/tmp1erm5k6g']
[2025-05-30T01:23:22.205+0000] {standard_task_runner.py:105} INFO - Job 113: Subtask execute_task
[2025-05-30T01:23:22.309+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:22.584+0000] {task_command.py:467} INFO - Running <TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T00:56:00+00:00 [running]> on host 31d54648d280
[2025-05-30T01:23:22.601+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:23.197+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flag_check_mysql_dag' AIRFLOW_CTX_TASK_ID='execute_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T00:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T00:56:00+00:00'
[2025-05-30T01:23:23.199+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T01:23:23.286+0000] {logging_mixin.py:190} INFO - Flag detectado, ejecutando tarea...
[2025-05-30T01:23:23.288+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-30T01:23:23.339+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T01:23:23.339+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flag_check_mysql_dag, task_id=execute_task, run_id=scheduled__2025-05-29T00:56:00+00:00, execution_date=20250529T005600, start_date=20250530T012321, end_date=20250530T012323
[2025-05-30T01:23:23.533+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-30T01:23:23.615+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
